app/src/main/java/ru/suslovalex/androidacademyapp/FragmentMoviesList.kt

```diff
@@ -1,14 +1,20 @@
 package ru.suslovalex.androidacademyapp
 
-import android.content.Context
 import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import ru.suslovalex.androidacademyapp.adapters.AdapterMovies
+import ru.suslovalex.androidacademyapp.adapters.MovieItemClickListener
+import ru.suslovalex.androidacademyapp.repository.getListMovies
 
 class FragmentMoviesList: Fragment() {
 
+    private lateinit var adapterMovies: AdapterMovies
```

Comment:
@AleksandrAlex Same here

Reply:
I think in this case to put the movies by setter - the best way. Cause when the movies be update we will not need to rebuild  all adapter. Just set movies into it. Am I right?

Reply:
For this implementation of the adapter and these conditions constructor is better. For updating the list better to use `ListAdapter` with diff utils https://developer.android.com/reference/kotlin/androidx/recyclerview/widget/ListAdapter